test/integration/targets/plugin_loader/ignore_exts/library/demo.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python
```

Comment:
To really prove that this is working properly, you should probably also have something like an empty `demo.ini` in the same directory with this that would otherwise get picked up to prove that it kept looking.